Gruntfile.js

```diff
@@ -77,9 +84,17 @@ module.exports = function(grunt) {
           build: {
             files: ['app/**/*'],
             tasks: ['deploy']
-          },
-
-        }
+          }
+        },
+        html2js: {
+            options: {
+                base: 'app/components'
```

Comment:
please only use `app` and remove the components part. 
add the `components` path part to the directives.